src/components/controls/deprecated/look-controls.js

```diff
@@ -1,9 +1,12 @@
-var registerComponent = require('../core/component').registerComponent;
-var THREE = require('../lib/three');
+var registerComponent = require('../../../core/component').registerComponent;
+var THREE = require('../../../lib/three');
+var debug = require('../../../utils/debug');
```

Comment:
Slowly moving towards that pattern, since `AFRAME.utils` is public API, I wanna mimic that.